backend/src/impl/auth.py

```diff
@@ -144,7 +159,7 @@ def fetch_user_info(self, username: str | None = None, email: str | None = None)
         if not users:
             return None
         if len(users) > 1:
-            raise RuntimeError(f"user ID {username or email} is not unique")
+            raise RuntimeError(f"user ID {sub or email} is not unique")
```

Comment:
This message contains some sensitive information and will be exposed to the application log; better to be avoided

```suggestion
            raise RuntimeError(f"user sub or email is not unique")
```